structure.go

```diff
@@ -102,25 +102,26 @@ const (
    https://priv.example.com/fileSequence2682.ts
 */
 type MediaPlaylist struct {
-	TargetDuration float64
-	SeqNo          uint64 // EXT-X-MEDIA-SEQUENCE
-	Segments       []*MediaSegment
-	Args           string // optional arguments placed after URIs (URI?Args)
-	Iframe         bool   // EXT-X-I-FRAMES-ONLY
-	Closed         bool   // is this VOD (closed) or Live (sliding) playlist?
-	MediaType      MediaType
-	durationAsInt  bool // output durations as integers of floats?
-	keyformat      int
-	winsize        uint // max number of segments displayed in an encoded playlist; need set to zero for VOD playlists
-	capacity       uint // total capacity of slice used for the playlist
-	head           uint // head of FIFO, we add segments to head
-	tail           uint // tail of FIFO, we remove segments from tail
-	count          uint // number of segments added to the playlist
-	buf            bytes.Buffer
-	ver            uint8
-	Key            *Key // EXT-X-KEY is optional encryption key displayed before any segments (default key for the playlist)
-	Map            *Map // EXT-X-MAP is optional tag specifies how to obtain the Media Initialization Section (default map for the playlist)
-	WV             *WV  // Widevine related tags outside of M3U8 specs
+	TargetDuration   float64
+	SeqNo            uint64 // EXT-X-MEDIA-SEQUENCE
+	Segments         []*MediaSegment
+	Args             string // optional arguments placed after URIs (URI?Args)
+	Iframe           bool   // EXT-X-I-FRAMES-ONLY
+	Closed           bool   // is this VOD (closed) or Live (sliding) playlist?
+	MediaType        MediaType
+	DiscontinuitySeq uint32 // EXT-X-DISCONTINUITY-SEQUENCE
```

Comment:
```suggestion
	DiscontinuitySeq uint64 // EXT-X-DISCONTINUITY-SEQUENCE
```
I suggest declare this field as type of `uint64` according to [RFC 8216](https://tools.ietf.org/html/rfc8216#section-4.3.3.3):

> Its format is:
> #EXT-X-DISCONTINUITY-SEQUENCE:<number>
> where number is a decimal-integer.